lib/classlink_client/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module ClasslinkClient
-  VERSION = "0.1.4"
```

Comment:
can you remove the version update? the versioning/release process isn't related to this MR.